pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -712,6 +723,42 @@
       %^  return-static-data-on-duct  404  'text/html'
       (error-page 404 authenticated url.request ~)
     ==
+ ::  +handle-cache-req: respond with cached value, 404 or 500
+  ++  handle-cache-req
+    |=  [authenticated=? =request:http =path]
+    |^  ^-  (quip move server-state)
+    ?.  authenticated
+      (error-response 403 ~)
+    ?.  =(%'GET' method.request)
+      (error-response 405 "may only GET scries")
+    =/  cache-entries  (fall (~(get of cache.state) path) ~)
```

Comment:
Let's treat `~` cache-entries as a different case than there not being an entry present. (Equivalent to the `~` vs `[~ ~]` pattern in scries. `~` will denote that the content does not exist and has never existed, and `[~ ~]` denotes that the content has been deleted.